gnomADc.pm

```diff
@@ -15,78 +13,63 @@ See the License for the specific language governing permissions and
-
- To download the gnomad exomes coverage file in TSV format: 
-  wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/exomes/gnomad.exomes.coverage.summary.tsv.bgz --no-check-certificate
-
- The coverage summary files must be processed and Tabix indexed before
- use by this plugin. 
-
- The following steps are necessary to tabix the gnomad genomes coverage file :
- mv gnomad.genomes.coverage.summary.tsv.bgz gnomad.genomes.r2.1.gz
- gunzip gnomad.genomes.r2.1.gz
- sed '1s/.*/#&/'  gnomad.genomes.r2.1 > gnomad.genomes.tabbed.tsv
- bgzip gnomad.genomes.tabbed.tsv
- tabix -s 1 -b 2 -e 2 gnomad.genomes.tabbed.tsv.gz
-
- The following steps are neccessary to tabix the gnomad exomes coverage file :
- mv gnomad.exomes.coverage.summary.tsv.bgz gnomad.exomes.r2.1.gz
- gunzip gnomad.exomes.r2.1.gz
- sed '1s/.*/#&/'  gnomad.exomes.r2.1 > gnomad.exomes.tabbed.tsv
- bgzip gnomad.exomes.tabbed.tsv
- tabix -s 1 -b 2 -e 2 gnomad.exomes.tabbed.tsv.gz
-
- By default, the output field prefix is 'gnomAD'. However if the input file's
- basename is 'gnomADg' (genomes) or 'gnomADe' (exomes), then these values are
- used instead. This makes it possible to call the plugin twice and include
- both genome and exome coverage values in a single run. For example:
-
- ./vep -i variations.vcf --plugin gnomADc,/path/to/gnomADg.gz --plugin gnomADc,/path/to/gnomADe.gz
-
+ 
+   https://gnomad.broadinstitute.org/downloads
+To download the gnomad coverage file in TSV format:
+ for Assembly GRCh37: 
+  gnomad genomes:
+   wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/genomes/gnomad.genomes.coverage.summary.tsv.bgz --no-check-certificate
+  gnomad exomes:
+   wget https://storage.googleapis.com/gcp-public-data--gnomad/release/2.1/coverage/exomes/gnomad.exomes.coverage.summary.tsv.bgz --no-check-certificate
+  
+ for Assembly GRCh38: 
+  gnomad genomes: 
+   wget https://storage.googleapis.com/gcp-public-data--gnomad/release/3.0.1/coverage/genomes/gnomad.genomes.r3.0.1.coverage.summary.tsv.bgz --no-check-certificate 
+
+The following steps are neccessary before using the plugin
+ for Assembly GRCh37:
+  The following steps are necessary to tabix the gnomad genomes coverage file :
+   gunzip -c gnomad.genomes.coverage.summary.tsv.bgz | sed '1s/.*/#&/' > gnomad.genomes.tabbed.tsv
+   bgzip gnomad.genomes.tabbed.tsv
+   tabix -s 1 -b 2 -e 2 gnomad.genomes.tabbed.tsv.gz
+   
+   The following steps are neccessary to tabix the gnomad exomes coverage file :
+   gunzip -c gnomad.exomes.coverage.summary.tsv.bgz | sed '1s/.*/#&/' > gnomad.exomes.tabbed.tsv
+   bgzip gnomad.exomes.tabbed.tsv
+   tabix -s 1 -b 2 -e 2 gnomad.exomes.tabbed.tsv.gz
+ 
+ for Assembly GRCh38:
+  The following steps are necessary to tabix the gnomad genomes coverage file :
+   mv gnomad.genomes.r3.0.1.coverage.summary.tsv.bgz gnomad.r3.0.1.gz
```

Comment:
`mv gnomad.genomes.r3.0.1.coverage.summary.tsv.bgz gnomad.r3.0.1.gz`
gnomad.r3.0.1.gz is not used, this command can be deleted.